code/game/machinery/kitchen/smartfridge.dm

```diff
@@ -13,11 +15,9 @@
 	flags = NOREACT
 	var/icon_on = "smartfridge"
 	var/icon_off = "smartfridge-off"
-	var/item_quants = list()
-	//var/ispowered = 1 //starts powered
-	//var/isbroken = 0
-	//OBSOLETE: That's what the BROKEN and NOPOWER stat bitflags are for
+	var/list/datum/fridge_pile/piles = list()
```

Comment:
Where is the code that actually deletes these piles on deletion of the fridge itself?